docs/source/devguides/walkthroughs/Forward-Historian-Deployment.rst

```diff
@@ -0,0 +1,76 @@
+Forward Historian Deployment
+=============================
+
+This guide describes a simple setup where one Volttron instance collects
+data from a fake devices and sends to another instance . Lets consider the
+following example.
+
+We are going to create two VOLTTRON instances and send data from one VOLTTRON 
+instance running a fake driver(subscribing values from a fake device)and sending
+the values to the second VOLTTRON instance.
+
+VOLTTRON instance 1 forwards data to VOLTTRON instance 2
+--------------------------------------------------------
+
+VOLTTRON instance 1 
+~~~~~~~~~~~~~~~~~~~
+
+-  Vctl shutdown –platform (if the platform is already working)
```

Comment:
These unicode dashes should be removed. vctl does not have a capital v. All shell commands, code snippets, and string literals should be formatted with a monospace font. And should maybe get their own code block.

Reply:
Got it . Converted all the code snippets to monospacing